tools/regression_tests/src/web/assets/jquery.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v1.11.3 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
On Sun, Dec 13, 2015 at 1:43 PM, Pedro Côrte-Real <notifications@github.com>
wrote:

> In tools/regression_tests/src/web/assets/jquery.js
> <https://github.com/darktable-org/darktable/pull/1073#discussion_r47442950>
> :
>
> > @@ -0,0 +1,5 @@
> > +/*! jQuery v1.11.3 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
>
> It can but it's very unsafe and leads to issues if you don't have a
> network connection for some reason. We already ship a few others in
> data/js/ for the same reasons I assume.
>
Which leads to issues like this: https://redmine.darktable.org/issues/10767

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/darktable-org/darktable/pull/1073/files#r47442950>.
>